code/parachain/frame/pablo/src/lib.rs

```diff
@@ -969,6 +974,20 @@ pub mod pallet {
 		}
 	}
 
+	pub(crate) fn create_lpt_asset<T: Config>(nonce: u64) -> Result<T::AssetId, DispatchError> {
+		T::LPTokenFactory::create_local_asset(
+			T::PalletId::get().0,
+			nonce,
+			AssetInfo {
+				name: None,
+				symbol: None,
```

Comment:
NIT: Symbol can be standardized in consultation with product

Reply:
I intentionally left these blank so product could edit the name/symbol for any protocol asset we're creating. It ultimately seemed like a human decision instead of something to enforce with code and string processing on chain.